<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>660</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>16</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>16</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>16</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
